pkg/api/api.go

```diff
@@ -290,6 +290,7 @@ func (a *API) RegisterRuler(r *ruler.Ruler) {
 
 	// List all user rule groups
 	a.RegisterRoute("/ruler/rule_groups", http.HandlerFunc(r.ListAllRules), false, true, "GET")
+	a.RegisterRoute(path.Join(a.cfg.PrometheusHTTPPrefix, "/api/v1/status/buildinfo"), buildInfoHandler, false, true, "GET")
```

Comment:
Should be moved to `RegisterRulerAPI()`, which is where Prometheus-compatible APIs are expected to be registered.

Reply:
done in 1b3f2135fc1229b7c0b0cf51e182564a7b16bb23